.travis.yml

```diff
@@ -20,10 +20,6 @@ cache:
     - /home/travis/gopath/pkg
 
 script:
-  - sudo service mysql stop
-  - sudo service postgresql stop
-  - docker-compose up -d
```

Comment:
I think we should keep those. First two lines stop MySQL and PostgreSQL that are running so we don't accidentally use them in our tests. Third line (that could be replaced by `make test-env-up`) start Docker Compose in the background, so it starts while we install dep, check vendor/, etc. That speed ups build.